web/modules/custom/joinup_community_content/js/pinned_content.js

```diff
@@ -42,9 +42,10 @@
           return;
         }
 
+        $this.addClass('listing__card--corner');
         // It is needed for listing title padding if image doesn't exist.
         if (!$this.find('.listing__image').length) {
-          $this.addClass('listing__card--title-pin');
+          $this.addClass('listing__card--corner-title');
```

Comment:
I tried the scenario where there is no image, here's how the helper fixes this:
![tile_with_helper_class](https://user-images.githubusercontent.com/1923476/30592626-cad22e3e-9d47-11e7-9fa5-fc4b81f30381.png)

And this is how it looks without a helper:
![tile_with_no_helper_class](https://user-images.githubusercontent.com/1923476/30592631-d1404f6c-9d47-11e7-9c3b-bed8b69e73f9.png)

Are you sure this helper is actually helping for better visualization?

Reply:
In my opinion this helper is useful. Your title fits perfectly into the tiles, if you will add few more letters in first line the title will overlap the icon.